features/step_definitions/basic_steps.rb

```diff
@@ -15,15 +15,14 @@
 end
 
 Given('We have the following user:') do |table|
-  table.hashes.each do |_field|
-    create(:user)
+  table.hashes.each do |field|
+    FactoryBot.create(:user, field)
```

Comment:
Why do you revert to include this. The configuration to include FactoryBot methods was done in another PR

Reply:
I didn't really get your comment.
And also no idea what does underscore in `|_feild|` does. 
It was not creating the required users when debugging, so changed it, believing this is the simplest way to create users and recipes using FactoryBot.

Reply:
Got it. Done.